tests/phpunit/tests/post/wpGetAutoSave.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * @group post
+ */
+class Tests_Post_GetPostAutosave extends WP_UnitTestCase {
+
+	/**
+	 * Admin user ID.
+	 *
+	 * @var int
+	 */
+	protected static $admin_id;
+
+	/**
+	 * Editor user ID.
+	 *
+	 * @var int
+	 */
+	protected static $editor_id;
+
+	/**
+	 * Post ID.
+	 *
+	 * @var int
+	 */
+	protected static $post_id;
+
+	/**
+	 * Ticket #62658
+	 *
+	 * @see https://core.trac.wordpress.org/ticket/62658
+	 */
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$admin_id  = $factory->user->create( array( 'role' => 'administrator' ) );
+		self::$editor_id = $factory->user->create( array( 'role' => 'editor' ) );
+
+		wp_set_current_user( self::$admin_id );
+		self::$post_id = $factory->post->create( array( 'post_status' => 'publish' ) );
+	}
+
+	/**
+	 * Test when no autosave exists for a post.
+	 */
```

Comment:
```suggestion
	/**
	 * Test when no autosave exists for a post.
	 *
	 * @ticket 62658
	 */
```

Add `@ticket` annotation for unit test.